src/main.c

```diff
@@ -48,6 +49,7 @@ strings_t strings;
 cx_sha3_t global_sha3;
 
 uint8_t appState;
+uint16_t apdu_response_code;
```

Comment:
`main.c` doesn't feel like the right place for all the globals definitions.. But I think we don't have a better place currently
What about a `globals.c` file that would contain all misc globals instead of main ? Plus it would allow having a dedicated .h file for globals declarations

Reply:
Same as last comment.